sonar-plugin-api/src/main/java/org/sonar/api/batch/fs/internal/PathPattern.java

```diff
@@ -56,6 +65,10 @@ public static PathPattern create(String s) {
     return result;
   }
 
+  /**
+   * @deprecated since 6.6
+   */
+  @Deprecated
   private static class AbsolutePathPattern extends PathPattern {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)